addons/binding/org.openhab.binding.netatmo/ESH-INF/thing/channels.xml

```diff
@@ -173,18 +172,18 @@
 	</channel-type>
 
 	<channel-type id="co2">
-		<item-type>Number</item-type>
+		<item-type>Number:Dimensionless</item-type>
 		<label>CO2</label>
 		<description>Air Quality in ppm</description>
-		<state readOnly="true" pattern="%d ppm" />
+		<state readOnly="true" pattern="%d %unit%" />
 	</channel-type>
 
 	<channel-type id="noise">
-		<item-type>Number</item-type>
+		<item-type>Number:Dimensionless</item-type>
 		<label>Noise</label>
 		<description>Current Noise Level in dB</description>
```

Comment:
Is there a reason why some of the descriptions still contain the unit? The README.md also contain channel descriptions with a certain unit. New or custom measurement systems may get added in the future which may cause unforeseen unit conversions.

Reply:
I see @cweitkamp  had the same remark. :-)

Reply:
JUst forget to change that. Wait few minutes...